src/common/low_precision_transformations/include/low_precision/layer_transformation.hpp

```diff
@@ -281,6 +283,8 @@ class LP_TRANSFORMATIONS_API LayerTransformation : public ov::pass::MatcherPass
         std::vector<ov::element::Type> defaultPrecisions;
         // to support GPU workarround to keep Reshape and MatMul in FP32
         bool reshapeIgnorePerTensorQuantizationCheck;
+        // for MultiplyPartialTransformation to support Activations Scaling
+        bool useDefaultTransformation;
```

Comment:
1. Please correct a comment: now we use this flag not only for `MultiplyPartialTransformation`
2. We use this flag in 2 **eltwise**-related transformations. Could you please reflect it in the name?

Reply:
I renamed it as `scalingMode`. If you have any better name, please let me know. Thank you.